content/kubermatic/master/guides/cluster_templates/_index.en.md

```diff
@@ -0,0 +1,32 @@
++++
+title = "Cluster templates"
+date = 2021-08-02T14:07:15+02:00
+weight = 20
+
++++
+
+## Understanding templates
+Cluster templates are designed to standardize and simplify the creation of Kubernetes clusters. A cluster template is a
+reusable cluster template object. It guarantees that every cluster it provisions from the template is uniform and consistent
+in the way it is produced.
+
+A cluster template allows you to specify a provider, node layout, and configurations to materialize a cluster instance
+via Kubermatic API or UI.
+
+## Scope
+The cluster templates are accessible from different levels.
+ - global: (mange by admin user) visible for everyone
+ - project: accessible for the project users
+ - user: accessible for the template owner
+
+Template management is available from project level.
```

Comment:
May be explain the user and admin perspective a little more. 
How does the admin mark a template as global?
What about the other features (enforce updates for templates)?

Reply:
right now we don't have enforce update, I've added more about the global admin scope